src/graphql/queries.js

```diff
@@ -415,6 +418,48 @@ module.exports = {
     return cursor.exec();
   },
 
+  mostVotes: async (root, { filter, orderBy, limit, skip }, { db: { Votes } }) => {
+    const voterFilters = buildVoteFilters(filter);
```

Comment:
think we should only allow max 1 filter in the `filter`. so we don't data from two events. add a check to make sure the length === 1.

2 checks:
length === 1
topicAddress is there